packages/grafana-schema/src/raw/team/x/team_types.gen.ts

```diff
@@ -9,16 +9,16 @@
 // Run 'make gen-cue' from repository root to regenerate.
 
 export enum Permission {
-  Admin = 4,
-  Editor = 2,
-  Viewer = 1,
+  admin = 4,
```

Comment:
How impactful are these changes for plugins? Should we worry about the levitate report? Or do plugins not use these yet? (They're public so I guess any plugin dev could pick them up if they wanted / needed to.)

Reply:
I don't think this is used anywhere yet, but I've updated the type to be consistent with how a similar enum is defined on front end.

Reply:
> Should we worry about the levitate report?

This is interesting - i wonder if we could selectively configure levitate to automatically ignore things below a certain [maturity level](https://docs.google.com/document/d/1gPzSCQn-0Qg86PYpPcR-l_kTQ4Ba1wgnNLZFKPSe8yQ/edit#heading=h.480z6zsrjqh2)?